### ISSUE TITLE
Loss is in millions ...experience_replay()

### ISSUE THREAD

**Issue by arjuniyer1978** [Albert-Z-Guo/Deep-Reinforcement-Stock-Trading#5]
Hello Albert,
Please pardon this note as an issue, new to github, not sure where else i can ask a question.

I am trying to run your DQN model, and i find that the loss value reported is very high, not sure how to read the loss...
I have printed the following in the training.py file - 

print(e, loss, action_dict[action], reward, agent.balance, len(agent.inventory),len(agent.memory),agent.buffer_size)
            logger.info('Episode {}\tLoss: {:.2f}\tAction: {}\tReward: {:.2f}\tBalance: {:.2f}\tNumber of Stocks: {}'.format(e, loss, action_dict[action], reward, agent.balance, len(agent.inventory)))

![image](https://user-images.githubusercontent.com/27776149/94359129-d3ae4100-00c2-11eb-94de-b0d5a8a71eb6.png)

Could you please clarify about the loss ?

Thank You
Arjun

**Comment by Albert-Z-Guo** [comment]
Hi Arjun, most likely it's because the model is not converging. I just reduced the learning rate of DQN model.
